handler/strings.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 var (
-	archRe     = regexp.MustCompile(`(arm64|arm|386|amd64|x86_64|aarch64|32|64)`)
```

Comment:
Delete 32 and 64? Maybe you need to rebase?

Reply:
No no.. this is the proposed change. As it is today, [this case](https://github.com/ahmetb/kubectx/releases) for example  picks ppc64le for amd64. I imagine it just boils down to order. "32" and "64" are not discriminant enough given the current approach.